### DISCUSSION TITLE
Could not use https://cloud.walletconnect.com/app

### DISCUSSION THREAD

**Discussion by natsabari** [WalletConnect/walletconnect-monorepo#3288] [Developer Support]
With new preference popup not closing and now the app is not useable!
Any plans to fix this "TypeError: Failed to fetch"  error

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/2860620/0e3b06ff-7ca9-4362-bdd7-a80372ad5dad)

**Comment by natsabari** [comment]
This happens only from chrome and working fine in Edge browser

**Comment by iggyiccy** [comment]
I don't know who should I reach out to. I emailed contact@walletconnect.com previously about this matter but the bug remain unfixed for a month. 

Issue: 
- Get `TypeError: Fail to Fetch` when ever I click on the "Save Preference" button and this modal is blocking me from accessing the dashboard. Unable to perform any action cause the modal will just stay on top not allowing me to close it. 
- Console is getting this
 ```
Access to fetch at 'https://awvuaxlpezvhfkoznrox.supabase.c app: 1
o/rest/v1/consumers?uid=eg.b66cala7-dc4d-42e9-a363-c6739f30b609 from origin "https://cloud.walletconnect. com' has been blocked by
CORS policy: Method PATCH is not allowed by Access-Control-Allow-Methods in preflight response.
```

<img width="1381" alt="Screenshot 2023-09-27 at 10 04 16 am" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/44436279/e81b0537-4fd4-43bd-bfe0-70ba5c04e03e">
